fcrepo-serialization/src/test/java/org/fcrepo/camel/serialization/BinaryDisabledRouteTest.java

```diff
@@ -146,7 +147,7 @@ public void configure() throws Exception {
         });
         context.start();
         // this should be zero because writing binaries is disabled by default.
-        getMockEndpoint("mock:direct:binary_file").expectedMessageCount(0);
+        getMockEndpoint("mock:file:mock:direct:binary_file").expectedMessageCount(0);
```

Comment:
This is a strange formulation. I don't know what you're trying to mock here, but as it is, it's incorrect.